diff --git a/go/private/BUILD.bazel b/go/private/BUILD.bazel
index 51df4ed6e..0833b2c10 100644
--- a/go/private/BUILD.bazel
+++ b/go/private/BUILD.bazel
@@ -64,7 +64,7 @@ bzl_library(
         "@bazel_skylib//rules:common_settings",
         "@bazel_tools//tools/build_defs/cc:action_names.bzl",
         "@bazel_tools//tools/cpp:toolchain_utils.bzl",
-        "@io_bazel_rules_go_bazel_features//:features.bzl",
+        "@io_bazel_rules_go_bazel_features//:features",
         "@io_bazel_rules_nogo//:scope.bzl",
     ],
 )
diff --git a/go/private/polyfill_bazel_features.bzl b/go/private/polyfill_bazel_features.bzl
index 9d3c9203a..5481b6b0c 100644
--- a/go/private/polyfill_bazel_features.bzl
+++ b/go/private/polyfill_bazel_features.bzl
@@ -22,7 +22,14 @@ def _polyfill_bazel_features_impl(rctx):

     find_cpp_toolchain_has_mandatory_param = major_version_int > 6 or (major_version_int == 6 and minor_version_int >= 1)

-    rctx.file("BUILD.bazel", """exports_files(["features.bzl"])
+    rctx.file("BUILD.bazel", """
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+bzl_library(
+    name = "features",
+    srcs = ["features.bzl"],
+    visibility = ["//visibility:public"],
+)
+exports_files(["features.bzl"])
 """)
     rctx.file("features.bzl", _POLYFILL_BAZEL_FEATURES.format(
         find_cpp_toolchain_has_mandatory_param = repr(find_cpp_toolchain_has_mandatory_param),
