diff --git a/rules/BUILD.bazel b/rules/BUILD.bazel
index 7cc150e..861e240 100644
--- a/rules/BUILD.bazel
+++ b/rules/BUILD.bazel
@@ -23,7 +23,7 @@ go_test(
     name = "go_default_xtest",
     size = "small",
     srcs = ["generator_test.go"],
-    data = ["//go/tools/gazelle:testdata"],
+    data = ["//:testdata"],
     importpath = "github.com/bazelbuild/bazel-gazelle/rules_test",
     deps = [
         ":go_default_library",
