diff --git a/resolve/BUILD.bazel b/resolve/BUILD.bazel
index d23c6ee..d619a52 100644
--- a/resolve/BUILD.bazel
+++ b/resolve/BUILD.bazel
@@ -41,6 +41,6 @@ genrule(
     name = "std_package_list",
     srcs = ["@go_sdk//:packages.txt"],
     outs = ["std_package_list.go"],
-    cmd = "$(location //go/tools/gazelle/resolve/internal/gen_std_package_list) $< $@",
-    tools = ["//go/tools/gazelle/resolve/internal/gen_std_package_list"],
+    cmd = "$(location //resolve/internal/gen_std_package_list) $< $@",
+    tools = ["//resolve/internal/gen_std_package_list"],
 )
diff --git a/resolve/internal/gen_std_package_list/BUILD.bazel b/resolve/internal/gen_std_package_list/BUILD.bazel
index d96795e..aa39b03 100644
--- a/resolve/internal/gen_std_package_list/BUILD.bazel
+++ b/resolve/internal/gen_std_package_list/BUILD.bazel
@@ -4,7 +4,7 @@ go_binary(
     name = "gen_std_package_list",
     importpath = "github.com/bazelbuild/bazel-gazelle/resolve/internal/gen_std_package_list",
     library = ":go_default_library",
-    visibility = ["//go/tools/gazelle/resolve:__subpackages__"],
+    visibility = ["//resolve:__subpackages__"],
 )
 
 go_library(
