diff --git a/README.rst b/README.rst
index 0ed5ec8..085a632 100644
--- a/README.rst
+++ b/README.rst
@@ -2,7 +2,7 @@ Gazelle build file generator
 ============================
 
 .. All external links are here
-.. _go_repository: go/workspace.rst#go_repository
+.. _go_repository: https://github.com/bazelbuild/rules_go/blob/master/go/workspace.rst#go-repository
 
 .. role:: flag(code)
 .. role:: cmd(code)
@@ -16,7 +16,7 @@ be invoked directly in a project workspace, or it can be run on an external
 repository during the build as part of the go_repository_ rule.
 
 *Gazelle is under active development. Its interface and the rules it generates
-may change.*
+may change. Gazelle is not an official Google product.*
 
 .. contents:: **Contents** 
   :depth: 2
diff --git a/resolve/BUILD.bazel b/resolve/BUILD.bazel
index d23c6ee..d619a52 100644
--- a/resolve/BUILD.bazel
+++ b/resolve/BUILD.bazel
@@ -41,6 +41,6 @@ genrule(
     name = "std_package_list",
     srcs = ["@go_sdk//:packages.txt"],
     outs = ["std_package_list.go"],
-    cmd = "$(location //go/tools/gazelle/resolve/internal/gen_std_package_list) $< $@",
-    tools = ["//go/tools/gazelle/resolve/internal/gen_std_package_list"],
+    cmd = "$(location //resolve/internal/gen_std_package_list) $< $@",
+    tools = ["//resolve/internal/gen_std_package_list"],
 )
diff --git a/resolve/internal/gen_std_package_list/BUILD.bazel b/resolve/internal/gen_std_package_list/BUILD.bazel
index d96795e..aa39b03 100644
--- a/resolve/internal/gen_std_package_list/BUILD.bazel
+++ b/resolve/internal/gen_std_package_list/BUILD.bazel
@@ -4,7 +4,7 @@ go_binary(
     name = "gen_std_package_list",
     importpath = "github.com/bazelbuild/bazel-gazelle/resolve/internal/gen_std_package_list",
     library = ":go_default_library",
-    visibility = ["//go/tools/gazelle/resolve:__subpackages__"],
+    visibility = ["//resolve:__subpackages__"],
 )
 
 go_library(
diff --git a/rules/BUILD.bazel b/rules/BUILD.bazel
index 7cc150e..861e240 100644
--- a/rules/BUILD.bazel
+++ b/rules/BUILD.bazel
@@ -23,7 +23,7 @@ go_test(
     name = "go_default_xtest",
     size = "small",
     srcs = ["generator_test.go"],
-    data = ["//go/tools/gazelle:testdata"],
+    data = ["//:testdata"],
     importpath = "github.com/bazelbuild/bazel-gazelle/rules_test",
     deps = [
         ":go_default_library",
